crates/l3/appchain-core-contract-client/src/clients/mod.rs

```diff
@@ -0,0 +1 @@
+pub mod client;
```

Comment:
just put the content of `client.rs` here. No need for two files

Reply:
done